README.md

```diff
@@ -21,26 +21,22 @@ MaEVe is implemented in Go 1.20. Learn more about MaEVe and its existing compone
 
 ## Pre-requisites
 
-MaEVe runs in a set of Docker containers. This means you need to have `docker`, `docker-compose` and a docker daemon (e.g. docker desktop, `colima` or `rancher`) installed and running. 
+MaEVe runs in a set of Docker containers. This means you need to have `docker`, `docker-compose` and a docker daemon (e.g. docker desktop, `colima` or `rancher`) installed and running.
 
 ## Getting started
 
 To get the system up and running:
 
-1. Run the [./scripts/generate-tls-cert.sh](./scripts/generate-tls-cert.sh) script which will create a server
-certificate for the CSMS
-2. Run the [./scripts/get-ca-cert.sh](./scripts/get-ca-cert.sh) script with a token retrieved from 
-the [Hubject test environment](https://hubject.stoplight.io/docs/open-plugncharge/6bb8b3bc79c2e-authorization-token) 
-to retrieve the V2G root certificate and CPO Sub CA certificates - remember to put your token argument within quotes
-3. Run the [./scripts/run.sh](./scripts/run.sh) script with the same token to run all the required components - again, don't forget the quotes around the token
+1. `(cd config/certificates && make)`
+1. Run the [./scripts/run.sh](./scripts/run.sh) script with the same token to run all the required components - again, don't forget the quotes around the token
```

Comment:
Markdown is often read as text. Whilst a markdown renderer will correctly number things - this won't be true when reading text.